частично выполненное задание №8 и 10

vikzak opened the pull request:
оставил закомментированный код, в котором видно, что выполнялись и другие способы реализации того же меню.

vikzak committed:
частично выполненное задание №8 и 10

vikzak committed:
ДЗ №8 - доработки

vikzak committed:
ДЗ №8 - доработки 2

primus-anonymous commented on app/src/main/java/ru/gb/mynotes_ver2/ui/list/NoteListFragment.java:
```diff
@@ -50,27 +54,52 @@ public void onClick(Note note) {
+            public boolean onNavigationItemSelected(@NonNull MenuItem item) {
+                switch (item.getItemId()){
+                    case R.id.action_add:
+                        Toast.makeText(requireContext(), "add", Toast.LENGTH_SHORT).show();
```
BottomNavigationView применяется для основной навигации в приложение, как и nav drawer, но никак не для выполнения каких-то действий, как что-то удалить и что-то добавить.

vikzak committed:
ДЗ №8 - доработки 3 (сгруппированные заметки по датам)

vikzak committed:
ДЗ №8 - доработки 4
1) Добавил BottomNavigationView
2) теперь recyclerview.widget.RecyclerView не закрывает BottomNavigationView
3) группировка записей по дате (заметки и даты случайным образом генерируются)
4) как в уроке сделал случайное отображение ошибки, вывод пустого либо полного списка заметок

vikzak committed:
ДЗ №8 - доработки 5
1) реализована функция контекстного меню
2) реализованно удаление заметки через контекстное меню
---
1) Добавил BottomNavigationView
2) теперь recyclerview.widget.RecyclerView не закрывает BottomNavigationView
3) группировка записей по дате (заметки и даты случайным образом генерируются)
4) как в уроке сделал случайное отображение ошибки, вывод пустого либо полного списка заметок

vikzak committed:
ДЗ №8 - доработки 6
1) импорт содержимого заметок в форму редактирования
2) обновление кода + удаление мусора
---
1) реализована функция контекстного меню
2) реализованно удаление заметки через контекстное меню
---
1) Добавил BottomNavigationView
2) теперь recyclerview.widget.RecyclerView не закрывает BottomNavigationView
3) группировка записей по дате (заметки и даты случайным образом генерируются)
4) как в уроке сделал случайное отображение ошибки, вывод пустого либо полного списка заметок

vikzak committed:
ДЗ №8 - доработки 6
1) тестирование анимации
--
1) импорт содержимого заметок в форму редактирования
2) обновление кода + удаление мусора
---
1) реализована функция контекстного меню
2) реализованно удаление заметки через контекстное меню
---
1) Добавил BottomNavigationView
2) теперь recyclerview.widget.RecyclerView не закрывает BottomNavigationView
3) группировка записей по дате (заметки и даты случайным образом генерируются)
4) как в уроке сделал случайное отображение ошибки, вывод пустого либо полного списка заметок

vikzak committed:
ДЗ №8 - доработки 6
1) тестирование DateTimePicker. Всё остальное работает.
--
1) тестирование анимации
--
1) импорт содержимого заметок в форму редактирования
2) обновление кода + удаление мусора
---
1) реализована функция контекстного меню
2) реализованно удаление заметки через контекстное меню
---
1) Добавил BottomNavigationView
2) теперь recyclerview.widget.RecyclerView не закрывает BottomNavigationView
3) группировка записей по дате (заметки и даты случайным образом генерируются)
4) как в уроке сделал случайное отображение ошибки, вывод пустого либо полного списка заметок
